db/1databaseNotes.txt

```diff
@@ -1,24 +1,37 @@
-# DATABASE NOTES:
-(Martin Connor, 6/5/19);
+DATABASE NOTES:
 
+  (Martin Connor, 6/5/19);
+
+**********************************************************************
+
+TESTING:
+
+# SAVE A QUERY PLAN TO A FILE:
+
+psql -d bookings_portals -qAt -f explain.sql > .txt
+
+
+# TEST 'INNER JOIN' QUERY:
+
+EXPLAIN ANALYZE SELECT bookings.from_date,bookings.to_date FROM bookings INNER JOIN listings ON bookings.listing_id=listings.listing_id WHERE bookings.listing_id=3000000;
+
+
+**********************************************************************
+
+SEEDING:
 
 # GIVE MORE SPACE TO FILE:
-node --max-old-space-size=16384 seedCsvs.js
+node --max-old-space-size=16384 seedCSVs.js
 node --max-old-space-size=32768 seedNonRelationalFile.js
 
-# INSERT .SQL FILE FROM TERMINAL COMMAND LINE:
+
+# INSERT <--ACTUAL!--> .SQL FILE FROM TERMINAL COMMAND LINE:
 psql postgres < postgresSchema.sql
 
 
 # INSERT .CSV FILE INTO MONGODB FROM TERMINAL COMMAND LINE:
 
-mongoimport -d booking_test -c listingBookings --type json --file listingBookings_data.json
-
-# TESTING POSTGRES RETRIEVAL TIME:
-
-\timing
-
-EXPLAIN (ANALYZE) select * from listings where listing_id=10000000;
+mongoimport -d bookings_portals -c listingBookings --type json --file listingBookings_data.json
 
 
 # Refactored MongoDB Database that was seeded from JSON file:
```

Comment:
It's so great to write documentation. If for no one else, it really helps me read and remember what I write in my project.